console/src/components/Services/Data/DataActions.js

```diff
@@ -563,19 +563,22 @@ const getBulkColumnInfoFetchQuery = schema => {
     type: 'run_sql',
     args: {
       sql: fetchColumnTypesQuery,
+      read_only: true,
```

Comment:
don't you need `checkFeatureSupport` here?

Reply:
it's just the select along with with run_sql part that needs to be behind version check